src/undo.h

```diff
@@ -24,7 +24,7 @@ struct TxInUndoFormatter
 {
     template<typename Stream>
     void Ser(Stream &s, const Coin& txout) {
-        ::Serialize(s, VARINT(txout.nHeight * 2 + (txout.fCoinBase ? 1u : 0u)));
+        ::Serialize(s, VARINT(txout.nHeight * uint32_t{2} + (txout.fCoinBase ? 1u : 0u)));
```

Comment:
```suggestion
        ::Serialize(s, VARINT(txout.nHeight * uint32_t{2} + txout.fCoinBase));
```

It could make sense to change this for symmetry with the implementation in `Coin::Serialize`?

Reply:
This looks like a great idea. I'm double checking it.